package.json

```diff
@@ -2,12 +2,12 @@
   "name": "woocommerce-services",
   "version": "1.16.1",
   "scripts": {
-    "start": "cross-env NODE_ENV=development webpack-dev-server --hot --inline --watch --content-base dist --port 8085",
-    "dist": "cross-env NODE_ENV=production webpack",
+    "start": "cross-env NODE_ENV=development DISABLE_FEATURES=code-splitting CALYPSO_CLIENT=true webpack-dev-server --hot --inline --watch --content-base dist --port 8085",
+    "dist": "cross-env NODE_ENV=production DISABLE_FEATURES=code-splitting,wpcom-user-bootstrap CALYPSO_CLIENT=true webpack",
```

Comment:
Leaving `wpcom-user-bootstrap` on did spit a few warnings in the console, it has no relevance otherwise.